docs/readme.md

```diff
@@ -0,0 +1,88 @@
+# FABulous docs
+
+The upstream FABulous documentation is available at [https://fabulous.readthedocs.io](https://fabulous.readthedocs.io/en/latest/)
+
+
+## General 
+Our docs are build using [Sphinx](https://www.sphinx-doc.org/en/master).
+The documentation is written in [reStructuredText](https://docutils.sourceforge.io/rst.html) format.
+
+
+## Prerequisites
+To build the documentation, you should already have set up you virtual environment and installed the required packages to use FABulous,
+as described in the [README](../README.md). Make sure you have picked the right FABulous branch, you want to build the documentation for.
+
+First source your virtual environment:
+```bash
+$ source venv/bin/activate
+```
+
+Then navigate to the `docs` directory and install the required pacakges:
+```bash
+(venv) $ cd docs
+(vevn) $ pip install -r requirements.txt
+```
+
+
+## Building the documentation
+
+### HTML format
+
+To build the documentation in HTML format, run:
+
+```bash
+(venv) $ make html
+```
+This should create a `build/html/` directory path in the `docs` folder for the HTML documentation.
+
+Open it with your browser:
+
+```bash
+(venv) $ xdg-open build/html/index.html
+```
+
+### PDF format
+
+If you want to build the documentation in PDF format, you need to install additional packages.
+You need to have a working LaTeX installation on your system, you can find the needed packages
+in the [LaTeXBuilder sphinx documentation](https://www.sphinx-doc.org/en/master/usage/builders/index.html#sphinx.builders.latex.LaTeXBuilder).
+You also need to install [Imagesmagic](https://imagemagick.org/script/index.php), which you can install via `apt-get`:
+
+```bash
+$ sudo apt-get install imagemagick
+```
+
+To build the documentation in PDF format, run:
+
+```bash
+(venv) $ make latexpdf
+```
+
+This should create a `build/latex/` directory path in the `docs` folder for the PDF documentation.
```

Comment:
Maybe use either `folder` or `directory`, but not both? But I'm unsure about this, so just double check if this was intended.